lib/streamlit/runtime/websocket_session_manager.py

```diff
@@ -40,12 +40,6 @@ class WebsocketSessionManager(SessionManager):
     Active sessions are those with a currently active websocket connection. Inactive
     sessions are sessions without. Eventual cleanup of inactive sessions is a detail left
     to the specific SessionStorage that a WebsocketSessionManager is instantiated with.
-
-    NOTE: The name of this class isn't finalized since there technically isn't any
-    mention of websockets in its implementation, but a more generic name like
-    DefaultSessionManager also feels weird with the concept of an active session being
-    so closely tied to websocket lifecycle. We'll figure out a final name for this
-    before the feature branch goes into `develop`.
```

Comment:
I think(?) leaving this name as-is is fine given that something like `DefaultSessionManager` feels weird to me with the notion of an "active" session being so closely tied to a websocket connection for now.

No strong opinions on this, though, given that this name is will be trivial to change until we officially support configuring which `SessionManager` implementation a server is initialized with (and whether we even want to allow that is still an open question).